### ISSUE TITLE
Add handle-classpath for nrepl server

### ISSUE THREAD

**Issue by benjamin-asdf** [babashka/nbb#119]
```clojure

(def ops
  "Operations supported by the nrepl server"
  {:eval handle-eval
   :describe handle-describe
   :clone handle-clone
   :close handle-close
   :load-file handle-load-file})
   ```
   
   currently one of the issues I run into when connecting to the nrepl server with cider is that it depends on a "classpath" op. 
   Else it falls back to evaling "System/getProperty" which errs out. 
   
   
   -> I will try my hands at an implementation

**Comment by borkdude** [comment]
Should be fixed now right?

**Comment by benjamin-asdf** [comment]
fixed with 11a4db30939cfdb1de1f874b024b3e17eece7470
